js/src/logger.ts

```diff
@@ -460,16 +461,22 @@ function logFeedbackImpl(
     throw new Error(`source must be one of ${VALID_SOURCES}`);
   }
 
-  if (isEmpty(scores) && isEmpty(expected) && isEmpty(comment)) {
+  if (
```

Comment:
Should we also support logging tags through the rest API? I think we'd need to add them to the typespecs here: https://github.com/braintrustdata/braintrust-sdk/blob/28b5b2b6e88d6eeff6ed047c8a7959a78bb6d8f6/core/js/src/typespecs/api_types.ts#L574

Reply:
will do